components/BottomSheet/src/private/MDCSheetContainerView.m

```diff
@@ -312,27 +304,36 @@ - (BOOL)draggableView:(__unused MDCDraggableView *)view
   }
 }
 
+- (BOOL)canExtend {
+  return self.preferredSheetHeight != [self maximumSheetHeight];
```

Comment:
Maybe it's just the property names, but this is confusing. Shouldn't the ability to "extend" a sheet depend on the difference between the current height and the maximum, not the preferred height?

Reply:
This API is "can extend ever". Would renaming it help?

Reply:
Maybe just documentation for `@return YES if the sheet can be extended beyond its preferredHeight, else NO`?

Reply:
Ah, fair. Done.